definitions/features/foreman_proxy.rb

```diff
@@ -77,9 +93,12 @@ def backup_features(for_features)
   end
 
   def curl_cmd
-    "curl -w '\n%{http_code}' --silent -ks --cert #{cert_path}/client_cert.pem \
-      --key #{cert_path}/client_key.pem \
-      --cacert #{cert_path}/proxy_ca.pem https://$(hostname):9090"
+    ssl_cert = proxy_settings[:foreman_ssl_cert] || proxy_settings[:ssl_certificate]
+    ssl_key = proxy_settings[:foreman_ssl_key] || proxy_settings[:ssl_private_key]
+    ssl_ca = proxy_settings[:foreman_ssl_ca] || proxy_settings[:ssl_ca_file]
+
+    "curl -w '\n%{http_code}' -s --cert #{ssl_cert} --key #{ssl_key} --cacert #{ssl_ca} \
```

Comment:
Please use `''` around the arguments ` --cert '#{ssl_cert}' --key '#{ssl_key}' --cacert '#{ssl_ca}' `, or even better, use `format_shell_args` from https://github.com/theforeman/foreman_maintain/blob/77e624b77b3c197ac25a427098d84e1f163f6d49/lib/foreman_maintain/concerns/system_helpers.rb#L161 to generate this part of the command.

Reply:
Done with the changes and updated PR.